jupyter-notebooks/Subscriptions-API/Subscriptions_API_Quickstart.ipynb

```diff
@@ -5,51 +5,126 @@
+    "The Subscriptions API is Planet's next-gen data delivery API; subscribe to continuous delivery of imagery and metadata collections which meet your item filter criteria with a single API call. Subscriptions must be delivered or hosted on cloud storage and cannot be delivered to your local machine like deliveries from Data or Orders API's.\n",
+    "\n",
+    "\n",
+    "This tutorial will take a look at the functionality of the Subscription API with the Python SDK and how you might be able to incorporate it into your own workflows."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3c64700a-c3b9-474b-839d-885009e9a623",
+   "metadata": {},
+   "source": [
+    "## Requirements\n",
+    "\n",
+    "This tutorial assumes familiarity with the [Python](https://python.org) programming language throughout. Python modules used in this tutorial are:\n",
+    "* [IPython](https://ipython.org/) and [Jupyter](https://jupyter.org/)\n",
+    "* [planet-python-client](https://github.com/planetlabs/planet-client-python)\n",
+    "\n",
+    "You should also have an account on the Planet Platform. You will login to create an access token that authenticates your API key from the [account page](https://www.planet.com/account/).\n",
+    "\n",
+    "## Useful links \n",
+    "* [Planet SDK and Developer Resources](https://docs.planet.com/develop/sdks/#planet-sdk-for-python-and-cli/)\n",
+    "* [Planet Python Client Repo](https://github.com/planetlabs/planet-client-python)\n",
+    "* [Planet Subscriptions API Documentation](https://docs.planet.com/develop/apis/subscriptions/)"
    ]
   },
   {
    "cell_type": "markdown",
    "id": "add66e0a",
    "metadata": {},
    "source": [
-    "## Planet API Authentication & Cloud Delivery Credentials\n",
-    "The new SDK supports a couple different methods of authentication (and soon, will support OIDC / Okta token-based authentication as well). Read more in the docs here.\n",
+    "### Subscriptions Delivery \n",
     "\n",
-    "For this example, we'll use the getpass tool to pass in our API key as a string to the from_key method.\n",
+    "For Subscriptions, local delivery is not supported and subscriptions must be delivered to a cloud bucket or hosting service. For details on delivery schema for the supported cloud delivery platforms, see the [Subscriptions API Delivery Documentation](https://developers.planet.com/docs/subscriptions/delivery/).\n",
+    "<br>\n",
+    "<br>\n",
+    "\n",
+    "Valid Cloud platforms include:\n",
+    " - Google Cloud Storage (used in this tutorial)\n",
+    " - Google Earth Engine \n",
+    " - Amazon S3 (and other services that use the S3 API)\n",
```

Comment:
replace the parenthetical with "(and other S3-compatible cloud storage providers)", since that's how we refer to them in the docs